src/local_insight_engine/main.py

```
@@ -225,6 +381,120 @@ def export_existing_analysis(
+            report = AnalysisReport(
+                statistics=self.last_analysis_statistics
+            )
+
+            logger.debug("Analysis report generated successfully", {
+                "document_name": report.statistics.document_name,
+                "total_entities": report.statistics.merge_analysis.total_entities_after_merge,
+                "transmission_safe": report.statistics.compliance_report.transmission_safe,
+                "processing_time": f"{report.statistics.performance.total_processing_seconds:.2f}s"
+            })
+
+            return report
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)